src/System.Net.Security/src/System.Net.Security.csproj

```diff
@@ -154,17 +144,33 @@
     <Compile Include="$(CommonPath)\Interop\Windows\SChannel\Interop.SecurityStatus.cs">
       <Link>Interop\Windows\SChannel\Interop.SecurityStatus.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\Interop\Windows\SChannel\SecuritySafeHandles.cs">
+      <Link>Interop\Windows\SChannel\SecuritySafeHandles.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Windows\SChannel\StreamSizes.cs">
+      <Link>Interop\Windows\SChannel\StreamSizes.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Windows\SChannel\UnmanagedCertificateContext.cs">
+      <Link>Interop\Windows\SChannel\UnmanagedCertificateContext.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Windows\SChannel\SecurityPackageInfoClass.cs">
+      <Link>Interop\Windows\SChannel\SecurityPackageInfoClass.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Windows\SChannel\SslConnectionInfo.cs">
+      <Link>Interop\Windows\SChannel\SslConnectionInfo.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Windows\SChannel\SecSizes.cs">
+      <Link>Interop\Windows\SChannel\SecSizes.cs</Link>
```

Comment:
Thanks! Moving to secur32. 
These are defined in sspi.h and are common to all the SSPI modules.